Basic_Python_Diploma/YaDisk.py

```diff
@@ -129,56 +128,48 @@ def _create(_param):
         print("Текущий список папок:")
         self.print_all('folder')
 
-    def find_biggest(self, obj_type=None):
-        """For find_biggest you shall indicate 'obj_type'-argument: either 'file' or 'folder' """
-        lst = None
-        filename = None
-        if (obj_type is None) or (obj_type != 'file' and obj_type != 'folder'):
-            return print(self.find_biggest.__doc__)
-        else:
-            if obj_type == 'file':
-                lst = self.all_files
-                filename = "biggest_file_info.json"
-            elif obj_type == 'folder':
-                lst = self.all_folders
-                filename = "biggest_folder_info.json"
-        max_size_object = max(lst, key=lambda obj: obj.size)
-        with open(filename, "w", encoding="UTF-8") as f:
-            json.dump({max_size_object.name: max_size_object.size}, f)
-
-        return max_size_object
-
-    def delete(self, *objects):
+    def delete(self, objects):
         """метод удаляет папку или файл с яндекс.диска в корзину или навсегда"""
 
+        print(objects)
         print("\nДля удаления объекта(-ов) в Корзину просто нажмите Enter.\n"
               "Для полного удаления объекта(-ов) без возможности восстановления введите 1.")
         permanent = input("Введите ответ: ")
         perm_del = ''
         if permanent == "1":
             perm_del = {"permanently": "true"}
 
-        for obj in objects:
-            param = {'path': obj.path}
+        if type(objects) == list:
```

Comment:
```suggestion
        if isinstance(objects, list):
```

https://pythoner.name/isinstance-type

Reply:
Ну да, есть. А нам точно нужно умнее? У меня сюда приходит или YaFile, или YaFolder, или список из них.